st2common/st2common/util/crypto.py

```diff
@@ -57,28 +308,128 @@ def symmetric_encrypt(encrypt_key, plaintext):
     'Initialization Vector' per run and the IV is part of the output.
 
     :param encrypt_key: Symmetric AES key to use for encryption.
-    :type encrypt_key: :class:`keyczar.keys.AesKey`
+    :type encrypt_key: :class:`keyczar.keys.AESKey`
```

Comment:
This is close but not quite accurate. The class should be: `st2common.utils.crypto.AESKey`.

Reply:
Fixed.